website/files/models/base.py

```diff
@@ -12,7 +12,6 @@
 from modularodm.exceptions import NoResultsFound
 from dateutil.parser import parse as parse_date
 
-from api.base.utils import absolute_reverse
```

Comment:
These were all the ones loaded by the flask app, which required DRF, which required `api.base.settings` to be set. Not good when you are just trying to build the JS libraries. I didn't search for the rest because they didn't cause failures.